src/components/BoemlyFormControl/BoemlyFormControl.tsx

```diff
@@ -167,8 +176,29 @@ export const BoemlyFormControl: React.FC<BoemlyFormControlProps> = ({
           {rightAddonsOrElements.map((addonOrElement) => addonOrElement)}
         </InputGroup>
       )}
-      {helperText && <FormHelperText>{helperText}</FormHelperText>}
-      {errorMessage && (
+      {helperText && inputType === 'Slider' ? (
+        <FormHelperText
+          position={['absolute', 'absolute', null, 'relative']}
+          left={['0', '0', null, 'unset']}
+          top={['24', '24', null, 'unset']}
+        >
+          {helperText}
+        </FormHelperText>
+      ) : (
+        <FormHelperText>{helperText}</FormHelperText>
+      )}
+      {errorMessage && inputType === 'Slider' ? (
```

Comment:
```suggestion
      {errorMessage ?  inputType === 'Slider' ? (
```

otherwise the error icon will always display if `isInvalid`, even if the `errorMessage` is empty.